docs/load.md

```diff
@@ -0,0 +1,108 @@
+# load
+
+## Module Exports
+
+### isPlugin - tests a value to determine whether is a plugin (an object with a `load` method).
+
+```ts
+import { isPlugin } from 'src/load';
+
+// true
+isPlugin({
+	load() {}
+	normalize() {}
+});
+
+isPlugin(1); // false
+isPlugin([]); // false
+isPlugin([]); // false
+// false
+isPlugin({
+	observer() {}
+});
+
+```
+
+### load - dynamically loads a module or other resource.
+
+```ts
+import load, { useDefault } from 'src/load';
+
+// Load a single module
+load('mymodule').then(([ myModule ]: [ any ]) => {
+	// ...
+});
+
+// Load multiple modules
+load('namespace/first', 'namespace/second').then(([ first, second ]: [ any, any ]) => {
+	// ...
+});
+
+// Load modules with relative IDs as relative to the current module
+load(require, './first', './second').then(([ first, second ]: [ any, any ]) => {
+	// ...
+});
+
+// Automatically map modules to their default export
+load('namespace/first').then(useDefault).then(([ first ]: [ any ]) => {
+	// ...
+});
+
+// Load a custom resource via a plugin
+load(require, 'plugin!./template.html').then(([ html ]: [ string ]) => {
+	// ...
+});
+
+```
+
+#### Using Load Plugins
```

Comment:
Alternatively, it could be changed to just `Using Plugins` since it's already in the documentation for `load.ts`. Since `core/load` isn't really a loader, I tried to avoid using the word "loader" as much as possible.